test/ServiceBus.Tests/SlowConsumingTests/EHSlowConsumingTests.cs

```diff
@@ -0,0 +1,178 @@
+﻿using Microsoft.WindowsAzure.Storage.Table;
+using Orleans;
+using Orleans.AzureUtils;
+using Orleans.Providers.Streams.Common;
+using Orleans.Runtime;
+using Orleans.Runtime.Configuration;
+using Orleans.ServiceBus.Providers;
+using Orleans.Storage;
+using Orleans.Streams;
+using Orleans.TestingHost;
+using Orleans.TestingHost.Utils;
+using ServiceBus.Tests.TestStreamProviders;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TestExtensions;
+using UnitTests.GrainInterfaces;
+using UnitTests.Grains.ProgrammaticSubscribe;
+using Xunit;
+
+namespace ServiceBus.Tests.SlowConsumingTests
+{
+    [Collection(TestEnvironmentFixture.DefaultCollection)]
+    public class EHSlowConsumingTests : OrleansTestingBase, IClassFixture<EHSlowConsumingTests.Fixture>
```

Comment:
The test is also testing if the monitor config can be set correctly through EventHubProviderSettings class, so I'd prefer use real EventHubStreamProvider